### PR TITLE
Add CardCarousel component

### PR DESCRIPTION
**Pull request by glenngijsberts**
# Description

This PR will introduce a new component called CardCarousel. It makes it possible to display our Card component in a carousel. For now it displays 2 cards at a time on desktop and will display the cards in a horizontal scroll on Mobile. It's almost the same implementation as our current carousel in the main repo.

## Changes

- [x] Added CardCarousel component
- [x] Made it possible to give the Card component a footer (e.g. for testimonials etc)

## How to test

- Checkout this branch
- `$ yarn dev`
- `$ yarn storybook`
- Check it out, verify it's working as expected

## Screenshots

Screenshot 2021-04-15 at 17 56 47

### CODE REVIEW

File: src/components/CardCarousel/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  background-color: ${color.nova};
+  min-width: 320px;
+
+  &:hover,
+  &:focus {
+    background-color: ${color.nova};
+  }
+
+  @media ${device.tablet} {
+    /* min-width: 520px; */
```
Comment (Roohn) [inline src/components/CardCarousel/index.stories.tsx]: 🤔

**Review by Roohn — DISMISSED** [review]
Maybe some animation might be nice, but this looks good as well 👍🏼

**Comment by glenngijsberts** [comment]
> Maybe some animation might be nice, but this looks good as well 👍🏼

yeah agree, kept it simple for now but I will def. look into it at some point :)

**Pull request merged**